### ISSUE TITLE
Gutenberg 7.8.1, block style previews are no longer showing in the panel

### ISSUE THREAD

**Issue by carolinan** [WordPress/gutenberg#21542]
**Describe the bug**
When a custom block styles has been added with `register_block_style`, 
the preview in the block panel is blank.
Not only for the custom style, but all styles for that block.

If I deactivate Gutenberg, the style preview is showing again.

**To reproduce**
1. register a style, for example:
```
	register_block_style(
			'core/paragraph',
			array(
				'name'         => 'twentytwenty-border',
				'label'        => __( 'Border', 'twentytwenty' ),
				'inline_style' => '.is-style-twentytwenty-border {
					border: 2px solid currentColor;
					padding: 0.5rem;
				}',
			)
		);
```
2. Add or select an existing paragraph block.
3. Make sure the block panel is open and that the styles panel is expanded.

**Expected behavior**
I expect the preview to work.

**Screenshots**
Left: With Gutenberg. Right: Without Gutenberg,
![empty-preview](https://user-images.githubusercontent.com/7422055/79038057-adf5b400-7bd6-11ea-9818-9b5ca0119645.png)

**Desktop (please complete the following information):**
 - OS: Win 10
 - Browser: Chrome Version 81.0.4044.92
WordPress Version 5.4

**Comment by draganescu** [comment]
Hi there @carolinan I tested this by adding that style definition to the `functions.php` in TwentyTwenty.

Using the `master` branch of Gutenberg I ended up with this result:

<img width="979" alt="Screenshot 2020-04-13 at 18 11 30" src="https://user-images.githubusercontent.com/107534/79132142-7479a000-7db2-11ea-9f54-9fff43bf70fd.png">

... which seems to be working as expected but on WordPress 5.4 and Gutenberg 7.8.1 I ended up with this:

<img width="858" alt="Screenshot 2020-04-13 at 18 15 36" src="https://user-images.githubusercontent.com/107534/79132578-42b50900-7db3-11ea-81eb-7f891af9bbdc.png">

So it seems something fixed this issue in the meantime.

**Comment by carolinan** [comment]
Yes as far as I can tell it has been fixed for 7.9.0, (and 7.9.1) I can not reproduce it.

**Comment by carolinan** [comment]
I can not reproduce this, (now tested with 8.1),  so its about time to close it.
